Dockerfile.sle12-sp5

```diff
@@ -0,0 +1,76 @@
+# SLE12-SP5 is officially not available at the Docker Hub
+# because of some licensing issues, use openSUSE-42.3 as a replacement
+# as there is no openSUSE-42.5 release which would be the equivalent to SLES12-SP5.
+# It shares the same core packages and should be close enough to SLE12-SP5
+# for running the libyui builds.
+FROM opensuse:42.3
+
+# ensure the UTF-8 locales are set
+ENV LC_ALL=en_US.UTF-8
+
+# do not install the files marked as documentation (use "rpm --excludedocs")
+RUN sed -i -e "s/^.*rpm.install.excludedocs.*/rpm.install.excludedocs = yes/" /etc/zypp/zypp.conf
+
+# the NON-OSS repo is not needed, save the network bandwidth and some time (~5 seconds) for each refresh
+RUN zypper mr -d "NON OSS"
+
+# we need to install Ruby first to define the %{rb_default_ruby_abi} RPM macro
+# see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run
+# https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/build-cache
+# why we need "zypper clean -a" at the end
+# "curl" is needed for importing the GPG key by "rpm" command
+RUN zypper --non-interactive in --no-recommends \
+  curl ruby && zypper clean -a
+
+# Reuse the YaST SP5 repository, we do not want to maintain extra devel:libraries:libyui:SLE-12:SP5
+# subproject.
+#
+# Set a higher priority for the sle_12_sp5 repo to prefer the packages from
+# this repo even if they have a lower version than the original 42.3 packages.
+RUN zypper ar -f -p 95 https://download.opensuse.org/repositories/YaST:/SLE-12:/SP5/openSUSE_Leap_42.3/ \
+  sle12_sp5
+
+# import the YaST OBS GPG key
+RUN rpm --import https://build.opensuse.org/projects/YaST/public_key
+
+RUN RUBY_VERSION=`rpm --eval '%{rb_default_ruby_abi}'` && \
+  zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
```

Comment:
NP: I think the `--gpg-auto-import-keys` option is not needed anymore as we have already imported the GPG key with `rpm --import` call above.